app/src/main/java/com/bulletin/BulletinApp.kt

```diff
@@ -0,0 +1,29 @@
+package com.bulletin
+
+import android.app.Application
+import android.content.Context
+import android.content.pm.PackageManager
+import android.util.Log
+import com.bulletin.utilities.GsonHelper
+import com.google.gson.Gson
+
+class BulletinApp : Application() {
```

Comment:
Application class is not required for this.
Wherever context is required it needs to be passed in as parameter.

Reply:
I have same thinking but we need to discuss.